src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Values/CultureValue.cs

```diff
@@ -0,0 +1,43 @@
+using System.Globalization;
+using System.Text.Encodings.Web;
+using Fluid;
+using Fluid.Values;
+using OrchardCore.Localization;
+
+namespace OrchardCore.DisplayManagement.Liquid.Values;
+
+public class CultureValue : FluidValue
+{
+    private readonly CultureInfo _culture;
+
+    public CultureValue(CultureInfo culture) => _culture = culture;
+
+    public override FluidValues Type => FluidValues.Object;
+
+    public override bool Equals(FluidValue other)
+    {
+        throw new NotImplementedException();
```

Comment:
Maybe compare string representation with culture code? In case someone does `Culture == 'en-US'`

Reply:
Please test it though, not sure when `Equals` is invoked

Reply:
> Please test it though, not sure when Equals is invoked

I can write a unit test, but I'm not sure when it called either :)